miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/MiniAppApiImpl.kt

```diff
@@ -0,0 +1,9 @@
+package com.rakuten.tech.mobile.miniapp.api
+
+/**
+ * The interface for MiniApp components which require api interaction.
+ */
+internal interface MiniAppApiImpl {
```

Comment:
I also think this name doesn't explain the purpose very well, but I'm not sure what a good name would be. Maybe `UpdatableApiClient`? But then we might also add additional functionality other than just updating the client...

Reply:
Yeah let's pick `UpdatableApiClient`. I never good at naming.